src/guess.js

```diff
@@ -0,0 +1,50 @@
+import compareNumbers from './compare-numbers.js';
+import changeTC from './changeTC.js';
+
+
+//Reference DOM elements
+const button = document.getElementById('submit-guess');
+const numberGuess = document.getElementById('number-guess');
+const trialMessage = document.getElementById('high-low-right');
+const triesDisplay = document.getElementById('numberTries');
+const gameResult = document.getElementById('win-lose-result');
+
+const numberAnswer = Math.floor((Math.random() * 20) + 1);
+//console.log('Actual Answer: ' + numberAnswer);
+
+let tries = triesDisplay.textContent;
+
+button.addEventListener('click', () => {
+
+    let guess = numberGuess.value;
+
+    tries -= 1;
+    changeTC(triesDisplay, tries);
```

Comment:
writing this function was a really smart move! this shows that you understand what a function is and why you'd want to write one - awesome job!!